cluster/addons/fluentd-elasticsearch/README.md

```diff
@@ -72,7 +72,6 @@ avoid Fluentd pods scheduling there.
 [xPack]: https://www.elastic.co/products/x-pack
 [setupCreds]: https://www.elastic.co/guide/en/x-pack/current/setting-up-authentication.html#reset-built-in-user-passwords
 [fluentdCreds]: https://github.com/uken/fluent-plugin-elasticsearch#user-password-path-scheme-ssl_verify
-[fluentdEnvVar]: https://docs.fluentd.org/v0.12/articles/faq#how-can-i-use-environment-variables-to-configure-parameters-dynamically

```

Comment:
you need to remove the reference to this URL earlier in the file

Reply:
ok.done